tests/components/google_sheets/test_config_flow.py

```diff
@@ -21,6 +20,8 @@
 
 CLIENT_ID = "1234"
 CLIENT_SECRET = "5678"
+GOOGLE_AUTH_URI = "https://accounts.google.com/o/oauth2/v2/auth"
```

Comment:
Please define the constants in the integration instead, use them there and import to the tests here as well.

Reply:
Would this leave the tests deterministic in the unlikely event that the URLs were accidently changed?